scripts/helpers/storage_service.py

```diff
@@ -33,3 +38,35 @@ def lookup_ingest(ingest_id):
             return api_name, client, ingest_data
 
     raise IngestNotFound
+
+
+def lookup_ingest_id(environment, space, external_identifier, version):
+    if environment == "staging":
+        manifests_bucket = "wellcomecollection-vhs-storage-staging-manifests"
+    elif environment == "prod":
+        manifests_bucket = "wellcomecollection-vhs-storage-manifests"
+    else:
+        sys.exit(f"Unrecognised environment: {environment}")
+
+    s3 = create_aws_client_from_role_arn("s3", role_arn=READ_ONLY_ROLE_ARN)
+
+    matching_manifests = list(
+        list_s3_prefix(
+            s3,
+            bucket=manifests_bucket,
+            prefix=f"{space}/{external_identifier}/{version.replace('v', '')}/",
+        )
+    )
+
+    if len(matching_manifests) >= 1:
```

Comment:
I think empty lists are falsy.

Reply:
Good catch!

Reply:
Ah yeah, I remember, this was originally `if len(matching_manifests) == 1` and it would throw if it found more than one, but it's possible for us to create two manifests if the bag register (which creates the manifest) runs twice – but all the manifests should always have the same ingest ID, so picking any of them is fine.